sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1661,7 +1661,7 @@ private synchronized void goToUnknownAppsSettings(ReleaseDetails releaseDetails)
              * And a no U.I. activity of our own must finish in onCreate,
              * so it cannot receive a result.
              */
-            mForegroundActivity.startActivity(intent);
+            mContext.startActivity(intent);
```

Comment:
If starting an activity from a non activity context (this is the app context here), the intent requires FLAG_NEW_TASK, I'm surprised that it works like this.

Reply:
THank you for feedback. After discussion with the team, we decide to keep mForegroundActivity as a context provider and just check if it is not null.